pkgs/servers/heisenbridge/default.nix

```
@@ -1,5 +1,20 @@
 { lib, fetchFromGitHub, fetchpatch, python3 }:
-python3.pkgs.buildPythonApplication rec {
+
+let
+  python = python3.override {
+    packageOverrides = self: super: {
+      mautrix_0_16 = self.mautrix.overridePythonAttrs (oldAttrs: rec {
```

```suggestion
      mautrix = self.mautrix.overridePythonAttrs (oldAttrs: rec {
```

You can keep the name.